New destination connection tracking + tracking checkbox (#533)
akeneo/pim-helpcenter: content/md/connectivity/connection-dashboard.md

related: what-is-a-connection, how-to-connect-my-pim, manage-your-connections
---

# Overview

In the PIM, there is a dashboard to help you follow your [connections](what-is-a-connection.html) activity. It's called the `Connection dashboard`.

You can access it by clicking on the `Activity` menu, then on the `Connection dashboard` submenu. And here you are! :tada:

![Connection dashboard](../img/connection-dashboard.png)

::: info
This dashboard is still in its first version. In the future, it will show you many more indicators. Stay tuned! 🙂  
And also, don't hesitate to fill in the survey at the bottom of the dashboard page to vote for the next KPI you'd like to see in there.
:::

In this dashboard, for now, you can monitor both your source and destination connections.

# Source connection monitoring
For your source connections, you can easily follow in this dashboard:
- the number of products that were **created** by each of them over the last 7 days,
![Number of products created](../img/number-of-products-created.png)

> - the number of products that were **created** by all of them over the last 7 days,

Reviewer: ```suggestion
- the total number of **created** products over the last 7 days,
```

Author: I fear it will be less clear that it's the total for all the source connections.

Author: ```suggestion
- the number of products that were **created** over the last 7 days by all your sources,
```

![Number of products created](../img/number-of-products-created-all-connections.png)

> - the number of products that were **updated** by each of them over the last 7 days.

Reviewer: ```suggestion
- the number of products that were **updated**, for each source, over the last 7 days.
```

Author: ```suggestion
- the number of products that were **updated** over the last 7 days for each source.
```
To be similar to what you suggested in a previous comment.

![Number of products updated](../img/number-of-products-updated.png)

> - the number of products that were **updated** by all of them over the last 7 days.

Reviewer: ```suggestion
- the total number of **updated** products over the last 7 days.
```

Author: ```suggestion
- the number of products that were **updated** over the last 7 days by all your sources.
```

![Number of products updated](../img/number-of-products-updated-all-connections.png)

You can easily switch the source connection you want to monitor by clicking on the following dropdown.

![Source connection dropdown](../img/source-connection-dropdown.png)

# Destination connection monitoring

> For your destination connections, you can easily follow in this dashboard:

Reviewer: ```suggestion
In this dashboard, you can easily track the following indicators from your destination connections:
```

> - the number of products that were **sent** to each of your destinations over the last 7 days,

Reviewer: ```suggestion
- the number of products **sent** to each of your destinations over the last 7 days,
```

![Number of products created](../img/number-of-products-sent.png)

> - the number of products that were **sent** to all of your destinations over the last 7 days.

Reviewer: ```suggestion
- the total number of products **sent** to all of your destinations over the last 7 days.
```
